sway-core/src/semantic_analysis/ast_node/expression/typed_expression.rs

```diff
@@ -1506,10 +1506,11 @@ impl ty::TyExpression {
             }
             (true, None, None, None) => {
                 handler.append(module_probe_handler);
-                return Err(handler.emit_err(CompileError::Unimplemented(
-                    "this case is not yet implemented",
+                return Err(handler.emit_err(CompileError::Unimplemented {
+                    feature: "This case".to_string(),
```

Comment:
```suggestion
                    feature: "Path bindings of modules".to_string(),
```

"this case" refers to compiler code, which is not helpful to the developer.

Reply:
Thanks for pointing this out @jjcnn! I wouldn't call it a nit, it was rather a blissful ignorance from my side. It turned out in the end that it is one of those places that is semantically meaningless and `Unimplemented` wasn't the right error at all. The new error is added to properly cover this case.